### ISSUE TITLE
Rating slider is not a good pattern

### ISSUE THREAD

**Issue by JAWS-test** [w3c/aria-practices#2501]
In my opinion, the [rating slider](https://www.w3.org/WAI/ARIA/apg/example-index/slider/slider-rating.html) is not a pattern that we should recommend:
* if my blind colleague tells me that he operates the slider and I don't see any slider on the page, then it is confusing
* with the mouse I do not realize that I can select half values. If I click in the right half of a star, I won't notice that I can also click in the left half and give a different rating.
* Currently, it seems unclear whether or not the rating stars must meet the WCAG requirement for target size (see https://github.com/w3c/wcag/issues/2713). If yes, then they do not fulfill it

**Comment by JAWS-test** [comment]
[HTML specification](https://html.spec.whatwg.org/multipage/input.html#range-state-(type=range)) says:

> [The slider element] represents a control for setting the element's value to a string representing a number, but with the caveat that the exact value is not important, letting UAs provide a simpler interface than they do for the [Number](https://html.spec.whatwg.org/multipage/input.html#number-state-(type=number)) state.

Therefore a slider for exact values like in the rating example would be wrong

**Comment by jongund** [comment]
I agree that allowing half stars has user interface issues and should be avoided, but some people like building them, so we should provide a means to help them understand how to make them accessible.

Examples:
* https://blog.logrocket.com/build-a-half-star-rating-component-in-react-from-scratch/
* https://stackoverflow.com/questions/62096550/setting-up-half-icons-with-a-rating-system

**Comment by jongund** [comment]
@JAWS-test 
We discussed the example at a meting last month and agreed that it was not a good example in its current state.  We would like to have a rating example using the slider pattern, since we still feel that additional keystrokes for a larger number of rating options is useful for more efficiently selecting values.

The idea for changing the example is to change the 5 stars to 10 circles, mimicking bubble forms.

We would also add information on why the slider pattern maybe useful over the radio button pattern when the number of options is greater than some number N.

What do you think of this proposal?

**Comment by JAWS-test** [comment]
@jongund Yes, that sounds good. Thanks for the planned improvements
